src/atribucion/modulos/atribucion/infraestructura/schema/v1/eventos.py

```diff
@@ -0,0 +1,50 @@
+from pulsar.schema import *
+from atribucion.seedwork.infraestructura.schema.v1.eventos import EventoIntegracion
+
+class IdentidadUsuarioSchema(Record):
+    id_usuario = String()
+    id_anonimo = String()
```

Comment:
Faltaría ajustar este schema para evitar el error de schemas incompatibles respecto al que schema que se usa en tracking. https://github.com/MISO-projects/AlpesPartners/blob/51eda38b22ea6d6120a09d5ea0596fc9e769d7c5/src/tracking/modulos/interacciones/infraestructura/schema/v1/eventos.py#L14-L18

```suggestion
class IdentidadUsuarioSchema(Record):
    id_usuario = String()
    id_anonimo = String()
    direccion_ip = String()
    agente_usuario = String()
```

Al cambiar el schema puede que pulsar empiece a llorar, habría que reiniciarlo de ceros. Yo lo hago borrando la carpeta `data/bookkeeper` y `data/zookeeper` y luego bajar el contenedor y el volumen

```
docker-compose --profile pulsar down -v    
```

Ya con eso me andan perfecto los eventos